uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-k8s/src/main/java/org/uberfire/java/nio/fs/k8s/K8SFileSystemProvider.java

```diff
@@ -200,7 +200,8 @@ public boolean deleteIfExists(final Path pathIn, final DeleteOption... options)
     @Override
     public boolean isHidden(final Path pathIn) throws IllegalArgumentException, IOException, SecurityException {
         checkNotNull("path", pathIn);
-        return false;
+        return executeCloudFunction(client -> getFsObjCM(client, pathIn), KubernetesClient.class).map(K8SFileSystemUtils::isHidden)
```

Comment:
How about a simplified implementation like this:
1. Checking the file CM existence by calling: checkFileNotExistThenThrow(path, false)
2. Test with a condition like: assertThat(hidden.getFileName().toString().startsWith(K8S_FS_HIDDEN_FILE_INDICATOR)).isTrue()

Why it should be sufficient is that current nio2.k8s implementation does not real support the 'hidden' attribute rather than to work around the limitation which the value of k8s label can NOT start with a dot. Since the Unix file system, by convention, treats it as a hidden file, we could simply check the file name.

Reply:
sounds good to me, will adjust the PR